src/main/java/org/testcontainers/containers/TarantoolCartridgeContainer.java

```diff
@@ -106,6 +106,7 @@ public class TarantoolCartridgeContainer extends GenericContainer<TarantoolCartr
     private static final String ENV_TARANTOOL_DATADIR = "TARANTOOL_DATADIR";
     private static final String ENV_TARANTOOL_INSTANCES_FILE = "TARANTOOL_INSTANCES_FILE";
     private static final String ENV_TARANTOOL_CLUSTER_COOKIE = "TARANTOOL_CLUSTER_COOKIE";
+    private static final String healthyCmd = "return require('cartridge').is_healthy()";
```

Comment:
What will be if there is no cartridge rock?

I mean this line
```lua
return cartridge ~= nil ...
```

Reply:
Will recheck it

Reply:
If we don't have cartridge rock then we won't be able to build the image